composer.json

```diff
@@ -1,53 +1,62 @@
 {
     "name": "lmc/api-filter",
-    "description": "Parser/builder for filters from API query parameters.",
     "type": "library",
-    "autoload": {
-        "psr-4": {
-            "Lmc\\ApiFilter\\": "src/"
-        }
-    },
-    "autoload-dev": {
-        "psr-4": {
-            "Lmc\\ApiFilter\\": "tests/"
-        }
-    },
+    "description": "Parser/builder for filters from API query parameters.",
+    "license": "MIT",
     "require": {
         "php": "^7.3",
         "ext-mbstring": "*",
         "beberlei/assert": "^2.9 || ^3.0",
         "mf/collections-php": "^4.0"
     },
-    "suggest": {
-        "doctrine/orm": "To allow applying filters directly to QueryBuilder"
-    },
     "require-dev": {
         "doctrine/orm": "^2.7",
+        "ergebnis/composer-normalize": "^2.5",
         "lmc/coding-standard": "^2.0",
         "mockery/mockery": "^1.3",
         "php-coveralls/php-coveralls": "^2.2",
+        "php-parallel-lint/php-parallel-lint": "^1.2",
         "phpstan/extension-installer": "^1.0",
         "phpstan/phpstan": "^0.12.23",
         "phpstan/phpstan-beberlei-assert": "^0.12.2",
         "phpstan/phpstan-mockery": "^0.12.5",
         "phpunit/phpunit": "^7.5"
     },
+    "suggest": {
+        "doctrine/orm": "To allow applying filters directly to QueryBuilder"
+    },
     "config": {
         "sort-packages": true
     },
+    "autoload": {
+        "psr-4": {
+            "Lmc\\ApiFilter\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Lmc\\ApiFilter\\": "tests/"
+        }
+    },
     "scripts": {
         "all": [
+            "@lint",
             "@analyze",
             "@tests"
         ],
         "analyze": [
             "@cs",
             "@phpstan"
         ],
+        "cs": "vendor/bin/ecs check --ansi src/ tests/",
+        "fix": "vendor/bin/ecs check --ansi --clear-cache --fix src/ tests/",
+        "lint": [
+            "vendor/bin/parallel-lint -j 10 ./src ./tests",
+            "@composer validate",
+            "@composer normalize --dry-run"
```

Comment:
If I understand correctly, could you add `@composer normalize` to `fix` script?

Reply:
:+1: Good idea, updated